sdks/python/apache_beam/coders/coder_impl.pxd

```diff
@@ -25,11 +25,14 @@ cimport libc.stdint
 cimport libc.stdlib
 cimport libc.string
 
+cdef extern from "math.h":
+  libc.stdint.int64_t abs "llabs"(libc.stdint.int64_t)
+
 from .stream cimport InputStream, OutputStream
 from apache_beam.utils cimport windowed_value
 
 
-cdef object loads, dumps, create_InputStream, create_OutputStream, ByteCountingOutputStream, get_varint_size
+cdef object loads, dumps, create_InputStream, create_OutputStream, ByteCountingOutputStream, get_varint_size, past_unicode
```

Comment:
This is the declaration file. Basically, it says "hold these objects via C variables rather than in the module's dict." It's a bit clunky, but we have to do that here because the .py file is pure Python.